tests-api/spec/swagger_helper.rb

```diff
@@ -16,10 +16,10 @@
   # the root example_group in your specs, e.g. describe '...', swagger_doc: 'v2/swagger.json'
   config.swagger_docs = {
     "v1/swagger.yaml" => {
-      swagger: "2.0",
+      openapi: "3.0.1",
```

Comment:
why this change?

Reply:
lesson-api uses [swagger 2.0](https://lesson-api-test.herokuapp.com/swagger/?format=openapi)